homopolymer_fit/ising_fitter_homopolymer.ipynb

```diff
@@ -1095,21 +1146,21 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
+   "display_name": "Python 2",
    "language": "python",
-   "name": "python3"
+   "name": "python2"
   },
   "language_info": {
    "codemirror_mode": {
     "name": "ipython",
-    "version": 3
+    "version": 2
    },
    "file_extension": ".py",
    "mimetype": "text/x-python",
    "name": "python",
    "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.8.3"
+   "pygments_lexer": "ipython2",
```

Comment:
@barricklab-at-jhu - I'm not sure why these were reading as python2 here if you were working in the env when you made the changes. I'll merge for now and check them out myself to make sure they work